source/_components/notify.telegram.markdown

```diff
@@ -230,9 +311,21 @@ action:
         longitude: 117.22743
 ```
 
-Configuration variables:
-
-- **latitude** (*Required*): The latitude to send.
-- **longitude** (*Required*): The longitude to send.
-- **keyboard** (*Optional*): List of rows of commands, comma-separated, to make a custom keyboard.
-- **inline_keyboard** (*Optional*): List of rows of commands, comma-separated, to make a custom inline keyboard with buttons with associated callback data.
+{% configuration %}
+latitude:
+  description: The latitude to send.
+  required: true
+  type: [string, int]
```

Comment:
I believe it has to be `float`. But it is not yet supported.

Reply:
I thought so, but what do you have to fill in then?

Reply:
@klaasnicolaas Please use `float`, I'm going to add it to the list of allowed types for the documentation standards.